database/gdb/gdb_core_underlying.go

```diff
@@ -278,8 +278,9 @@ func (c *Core) DoCommit(ctx context.Context, in DoCommitInput) (out DoCommitOutp
 		c.writeSqlToLogger(ctx, sqlObj)
 	}
 	if err != nil && err != sql.ErrNoRows {
-		err = gerror.NewCodef(
+		err = gerror.WrapCodef(
 			gcode.CodeDbOperationError,
+			err,
 			"%s, %s",
 			err.Error(),
```

Comment:
If it uses `Wrap`, it then should remove the duplicated error string from its error message.

Reply:
Thanks. I'll make the modification.